.github/workflows/ci.yml

```diff
@@ -33,3 +33,15 @@ jobs:
           java-version: '11' # TODO read this from the `.java-version` file in the repository
           distribution: 'corretto'
       - run: ./scripts/ci
+
+      - uses: guardian/actions-riff-raff@v4
+        with:
+          projectName: devx::prism
```

Comment:
This matches [riffraff-platform](https://github.com/guardian/riffraff-platform/blob/e11cead4dadb2e9288fba0c26d6f8d3b73db538b/packages/common/src/access.ts#L22).